### ISSUE TITLE
Only part of elevation from lidar transferred into the course

### ISSUE THREAD

**Issue by MyronBedford** [chadrockey/TGC-Designer-Tools#108]
I'm wondering whether the files that I pulled didn't cover the entire course, since the files I pulled were from a singular project to map out  the whole city, (Canada)

**Comment by chadrockey** [comment]
What does the screenshot overview of the course and features show in the tool?  The terrain is the brown area, is some missing?

Any messages about a file being skipped?  Are all the files in your course having similar names?


You’re likely correct that one is missing, usually there will be a missing square or very straight lines where it’s just not there.

**Comment by chadrockey** [comment]
Thank you for moving your debug to a new issue.  The other issue is still pending Mapquest resolution.

**Comment by MyronBedford** [comment]
The OSM shows up all throughout but there is a black square that covers part of the course but it's a straight line so it definitely seems like I just didn't get all the files for the entire course.
All the files I have do have similar names and the section that shows up outside of the black square the lidar is just fine

**Comment by chadrockey** [comment]
Yes looks like you’ve missed a file.  You can always add a few extra and as long as you mask, it won’t affect your course.

**Comment by MyronBedford** [comment]
I've haven't been editing the mask since if I save it as the same name it doesn't accept it. I personally don't mind editing some of it inside of TGC 2019

**Comment by chadrockey** [comment]
You can’t remove terrain in game that the tool paints so the mask is critically important to save your plant meter.  Be sure to use a recommended paint tool, you have a mask.png and a mask.somethingelse, but it’s hard to tell because windows hides extensions now.

**Comment by MyronBedford** [comment]
It just asks me whether I want to replace the old mask.png
